plugins/modules/rabbitmq_policy.py

```diff
@@ -227,13 +435,25 @@ def main():
         priority=dict(default='0'),
         node=dict(default='rabbit'),
         state=dict(default='present', choices=['present', 'absent']),
+        # API Params.
+        login_user=dict(type="str", no_log=True),
+        login_password=dict(type="str", no_log=True),
+        login_host=dict(type="str"),
+        login_port=dict(type="str", default="15672"),
+        login_protocol=dict(type="str", default="http", choices=["http", "https"]),
+        ca_cert=dict(type="path"),
+        client_cert=dict(type="path"),
+        client_key=dict(type="path"),
     )
 
     module = AnsibleModule(
         argument_spec=arg_spec,
         supports_check_mode=True
     )
 
+    if not HAS_REQUESTS:
+        module.fail_json(msg=missing_required_lib("requests"), exception=REQUESTS_IMP_ERR)
```

Comment:
```suggestion
    if not HAS_REQUESTS:
        module.fail_json(msg=missing_required_lib("requests"), exception=REQUESTS_IMP_ERR)
```
@dmbuck32 should this be called **only** when one of those API login related options are passed? Maybe I miss something? I thought the module will use CLI by default, but now it'll always fail when `requests` isn't installed, correct?

Reply:
Yes that's correct. I'm intentionally requiring the `requests` library because this is  consistent with the other modules in this collection, as seen below:

https://github.com/ansible-collections/community.rabbitmq/blob/c2751205336ffdf932add6b8a381d74471b3b0f4/plugins/modules/rabbitmq_queue.py#L175-L177

https://github.com/ansible-collections/community.rabbitmq/blob/c2751205336ffdf932add6b8a381d74471b3b0f4/plugins/modules/rabbitmq_binding.py#L302-L304

https://github.com/ansible-collections/community.rabbitmq/blob/c2751205336ffdf932add6b8a381d74471b3b0f4/plugins/modules/rabbitmq_exchange.py#L131-L133

Except for the `rabbitmq_vhost` module, which falls back to the rabbitmqctl CLI:
https://github.com/ansible-collections/community.rabbitmq/blob/c2751205336ffdf932add6b8a381d74471b3b0f4/plugins/modules/rabbitmq_vhost.py#L311-L314

However I've created a PR (#207) to update the `rabbitmq_vhost` module to follow the same logic as the other modules.